### ISSUE TITLE
Model output with nan - check precision points, possible gradient exploding

### ISSUE THREAD

**Issue by jordancaraballo** [nasa-nccs-hpda/above-shrubs#3]
2023-05-01 13:27:44.691659: I tensorflow/compiler/jit/xla_compilation_cache.cc:480] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.
150/731 [=====>........................] - ETA: 1:33 - loss: 0.2067 - mean_squared_error: 0.1013 - root_mean_squared_error: 0.3182 - mean_absolute_error: 0.2065 - r_square: -2.7892Batch 150: Invalid l731/731 [==============================] - 70s 62ms/step - loss: nan - mean_squared_error: nan - root_mean_squared_error: nan - mean_absolute_error: nan - r_square: nan - val_loss: nan - val_mean_squared_error: nan - val_root_mean_squared_error: nan - val_mean_absolute_error: nan - val_r_square: nan - lr: 1.0000e-05

**Comment by jordancaraballo** [comment]
The new version of tensorflow keeps outputting NaN, we are disabling mixed precision for now until the fix the bug.
